### PR TITLE
Feature/en 14 make two buttons for answers

### PR DESCRIPTION
**Pull request by A-Romany**

### CODE REVIEW

File: src/main/java/org/agbrothers/englishbot/constant/MessageLabel.java [hunk 1]
```diff
@@ -8,5 +8,8 @@ private MessageLabel() {
     public static final String NEXT_LESSON = ",\nдля продовження навчання оберіть урок:";
     public static final String CORRECT_ANSWER = "Ви відповіли правильно! \n";
     public static final String INCORRECT_ANSWER = "Відповідь не вірна! Правильна відповідь - ";
-
+    public static final String NEWLINE = "\n";
+    public static final String PROBEL = " ";
```
Comment (AgienkoSergiy) [inline src/main/java/org/agbrothers/englishbot/constant/MessageLabel.java]: PROBEL? :) English please

File: src/main/java/org/agbrothers/englishbot/constant/MessageLabel.java [hunk 2]
```diff
@@ -8,5 +8,8 @@ private MessageLabel() {
...
+    public static final String FROM = "із";
```
Comment (AgienkoSergiy) [inline src/main/java/org/agbrothers/englishbot/constant/MessageLabel.java]: I suppose there are no so many usages of this string in code, hence it does'nt need to be constant

File: src/main/resources/application.yml [hunk 3]
```diff
@@ -16,7 +16,7 @@ spring:
 telegram:
   token: 1153088687:AAGlMcZ6W2qvE_YaWlYkXTkg9W2Yg8XZL0U
   name: English bot
-  bot.server.url:    https://1a34c4122e00.ngrok.io
+  bot.server.url: https://c8340a23baed.ngrok.io
```
Comment (AgienkoSergiy) [inline src/main/resources/application.yml]: exclude this file from commit please

**Review by AgienkoSergiy — CHANGES_REQUESTED** [review]

File: src/main/resources/application.yml (hunk 3 above)
Comment (A-Romany) [reply src/main/resources/application.yml]: ok

**Review by AgienkoSergiy — APPROVED** [review]

**Pull request merged**